connector/src/main/java/org.apache.flink/connector/nebula/connection/NebulaClientOptions.java

```diff
@@ -34,18 +32,21 @@
 
     private final boolean enableMetaSSL;
 
+    private final boolean enableStorageSSL;
+
     private final SSLSighType sslSighType;
 
-    private final CASignedSSLParam caSignParam;
+    private final CASignParams caSSLSignParams;
 
-    private final SelfSignedSSLParam selfSignParam;
+    private final SelfSignParams selfSignParams;
```

Comment:
one with SSL and another with not ?

Reply:
I'll fix the caSSLSignParams to caSignParams.